libs/labelbox/src/labelbox/data/serialization/ndjson/label.py

```diff
@@ -60,143 +60,19 @@
 class NDLabel(BaseModel):
     annotations: AnnotationType
 
-    class _Relationship(BaseModel):
```

Comment:
This stuff was used when we use to deserialized so not needed anymore also is very complicated